helix-term/src/args.rs

```diff
@@ -17,6 +17,8 @@ pub struct Args {
     pub log_file: Option<PathBuf>,
     pub config_file: Option<PathBuf>,
     pub files: Vec<(PathBuf, Position)>,
+    pub line_num: u32,
```

Comment:
This field is never accessed. A minor naming nit but it should probably be named `line_number` for clarity